### ISSUE TITLE
Game bar pops up

### ISSUE THREAD

**Issue by Perlence** [sdias/win-10-virtual-desktop-enhancer#57]
When I use hotkeys to switch to a desktop by its number, Game bar pops up and says, "Do you want to open Game bar?" Sometimes it disappears right away, sometimes it stays for a long time. I tried to disable tooltips, but it didn't help. Disabling Game bar all together actually works, but it's kind of brute solution.

**Comment by GioBonvi** [comment]
I cannot replicate this on my computer, however I have observed some behaviours which could be related.

It looks like `Ctrl + Win + Number` is a default Windows keybinding for "Start the nth program pinned in the taskbar: if that program is already running minimize/maximize its window".

My guess would be that you have some application pinned on the taskbar which causes the Game bar to fire up or something similar.  
Another option could be that Windows is considering this program to be a game, but this is quite a crazy hypothesis...

Does the problem happen with every number or only specific numbers? Are you using the default keybindings (`Ctrl + Win`) or some other customized ones? Lastly have you modified any Game Bar settings in the Windows settings app?

Thank you for your time.

**Comment by Perlence** [comment]
Actually I remapped SwitchDesktop modifier to <kbd>Alt</kbd>. This happens not only on my PC, but also in a newly imported Windows 10 Preview (15.15063) VM provided by [modern.ie](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/).

**Comment by sdias** [comment]
@Perlence Is there any chance that the keyboard shortcut to show the Game Bar is conflicting with one of the keyboard shortcuts in this app? Apart from that and what @GioBonvi has said, I can't think of any other reason it would do this.

**Comment by Perlence** [comment]
It looks like this only happens when `SwitchDesktop=Alt`. The bar appears on windows of Edge, Firefox and foobar2000, but not on Total Commander and Sublime Text.

**Comment by GioBonvi** [comment]
@Perlence do you mean that the bug only appears when switching from a desktop while having one of those programs focused? Or when  landing on a desktop with that program?

**Comment by Perlence** [comment]
So, it appears when an app is run as a simple user and its window has focus. If app is run as administrator the bar does not pop up no matter the focus.

**Comment by GioBonvi** [comment]
@sdias @Perlence 
I was able to replicate this behaviour on my machine (Build 15063.296).

This is what I observed:
 - the bug consists in the Windows Game Bar appearing even when applications are quite obviously not games as reported by @Perlence 
 - disabling the Windows Game Bar in the Settings app (first option in the Game settings page) prevents the bug from happening as reported by @Perlence 
 - the bug only happens when landing on an application (nothing ever happens if you switch to an empty desktop)
 - I was able to replicate the bug when landing on **any** application I tested except Windows Explorer (I tried Windows Explorer, Chrome, cmd.exe, Firefox, Edge, Spotify, Notepad++ and Teamviewer)
 - the bug happens both when switching to a desktop by its number and by arrow keys (next/previous desktop)
 - the bug seems to appear only if the `Alt` key is used in any keybinding, even if used in combination with other keys (tested with `SwitchDesktop` and `MoveWindowToDesktop`)
 - particular case: if `SwitchDesktop = Ctrl, Alt` the bug always happens except if the `Alt` key is released before the `Ctrl` key
 - changing `Alt` to `LAlt` does not influence the bug: I cannot test `RAlt` as I lack that key on my keyboard and do not have any external keyboard available
 - removing the Toasts did not influence the bug, as reported by @Perlence (I tested the script with all references to Tooltips commented out and it still happened)

To sum up: it looks like that using the `Alt` key in any of the main keybindings causes the Game Bar to pop up on any program.

The only workaraounds available at the moment are:
 - disabling the game bar completely
 - not using the `Alt` key in any keybinding

I will try to look around both in the code and on the internet for any possible conflict between the Alt key and the Game Bar later this weekend.

**Comment by vikonen** [comment]
I encountered this bug as well but I'm using `Win` key as modifier instead of `Alt` so I don't think this is related to only using `Alt` in keybindings.

I ended up disabling the game bar since I don't need it.

**Comment by Perlence** [comment]
@GioBonvi Great summary!

**Comment by GioBonvi** [comment]
@vikonen 

> I encountered this bug as well but I'm using Win key as modifier

Thank you for the info: updated my observations accordingly.

**Comment by sdias** [comment]
@GioBonvi I second @Perlence, great job in pinning this down.
Some weird interaction seems to be going on, and I'm not sure we will be able to fix it. Maybe we should consider a "Known Issues" section in the documentation.

**Comment by GioBonvi** [comment]
@sdias I agree: there are also some old issues that could be included.

**Comment by martinb3** [comment]
Hi folks -- just ran into this but I have the game bar disabled already. It seems that Task Manager in Windows 10 Creators Update (Fall 2018) also wants to capture CTRL-ALT-<Left/Right arrow>, specifically if it gains focus while the "Performance" tab is open, and it gets focused during desktop switching. Has anyone seen that or been able to reproduce it?
